### PR TITLE
fix(button, split-button, avatar): change condensed & super-condensed to 4px border-radius (VIV-1464)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/button/README.md [hunk 1]
```diff
@@ -86,6 +86,22 @@ Use the `size` attribute to set the button's to one of the predefined block size
...
+<vwc-button stacked icon='message-sent-line' appearance='filled' label='super-condensed' size='super-condensed'></vwc-button>
+<vwc-button stacked icon='message-sent-line' appearance='filled' label='condensed' size='condensed'></vwc-button>
+<vwc-button stacked icon='message-sent-line' appearance='filled' label='normal' size='normal'></vwc-button>
+<vwc-button stacked icon='message-sent-line' appearance='filled' label='expanded' size='expanded'></vwc-button>
+<hr>
+<vwc-button stacked icon='message-sent-line' shape='pill' appearance='filled' label='super-condensed' size='super-condensed'></vwc-button>
+<vwc-button stacked icon='message-sent-line' shape='pill' appearance='filled' label='condensed' size='condensed'></vwc-button>
+<vwc-button stacked icon='message-sent-line' shape='pill' appearance='filled' label='normal' size='normal'></vwc-button>
+<vwc-button stacked icon='message-sent-line' shape='pill' appearance='filled' label='expanded' size='expanded'></vwc-button>
+
```
Comment (rachelbt) [inline libs/components/src/lib/button/README.md:104]:
```suggestion
```

File: libs/components/src/lib/avatar/README.md
Comment (RichardHelm) [inline libs/components/src/lib/avatar/README.md:1]: Why did you change this? Prevents us from seeing the border radius?

Comment (rachelbt) [reply libs/components/src/lib/avatar/README.md:1]: yes. the preview was of 2 sets - one with initials one with icon but both where pill shape

Comment (RichardHelm) [reply libs/components/src/lib/avatar/README.md:1]: Ah, I was confused

**Review by RichardHelm — APPROVED** [review]
